test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -41,22 +41,32 @@
 
 @tags('multisched')
 class TestSchedulerInterface(TestInterfaces):
+
     """
     Test suite to test different scheduler interfaces
     """
 
     def setUp(self):
         # delete all schedulers except default
-        self.server.manager(MGR_CMD_DELETE, SCHED, id="@default")
+        self.server.manager(MGR_CMD_DELETE,
+                            SCHED,
+                            id="@default")
```

Comment:
I'll make this comment again since I didn't get a good answer to it.  This should be in revert_to_defaults().  This is analogous to PTL removing all queues.  PTL should remove all schedulers (except the default) between tests.  It shouldn't be up to the test itself to do it.